src/extension/common/components/copyButton.re

```diff
@@ -1,4 +1,10 @@
-let createElement ::label="copy" ::text ::onCopy ::style=? children::_ () =>
-  <CopyToClipboard text onCopy>
-    <span className="copy-button" style=?style> (ReactRe.stringToElement label) </span>
-  </CopyToClipboard>;
+
+let component = ReasonReact.statelessComponent "CopyButton";
+let make ::label="copy" ::text ::onCopy ::style=? _ => {
+  ...component,
+
+  render: fun _ _ =>
+    <CopyToClipboard text onCopy>
+      <span className="copy-button" style=?style> (ReactRe.stringToElement label) </span>
+    </CopyToClipboard>
+};
```

Comment:
I don't think you can, which really is a shame. The JSX is turned into this now: `ReasonReact.element (CopyToClipboard.make text::text onCopy::onCopy [|ReactDOMRe.createELement "span" ...|])`. It's not a direct function call anymore.